_includes/nav.html

```diff
@@ -1,57 +1,28 @@
-<nav class="navbar is-transparent">
-    <div class="navbar-brand">
-        <a class="navbar-item" href="/">
-            <img src="../images/rubynepal.png" alt="Ruby Nepal: The Official Ruby Community">
-        </a>
-        <div class="navbar-burger burger" data-target="navbarExampleTransparentExample">
-            <span></span>
-            <span></span>
-            <span></span>
-        </div>
+<nav class="navbar is-transparent rbnw-nav">
+  <div class="navbar-brand">
+    <a class="navbar-item rbnw-nav-logo" href="/">
```

Comment:
There shouldn't be red underline for logo

Reply:
Resolved :)